### PR TITLE
AuditReader: Use interface typehint for entityManager

### PR DESCRIPTION
**Pull request by TomasVotruba**
Ref #149

### CODE REVIEW

**Comment by DavidBadura** [comment]
thank you :-)

**Comment by TomasVotruba** [comment]
Thanks!

File: src/SimpleThings/EntityAudit/AuditReader.php [hunk 1]
```diff
@@ -25,6 +25,7 @@
 
 use Doctrine\DBAL\Types\Type;
 use Doctrine\ORM\EntityManager;
```
Comment (OskarStark) [inline src/SimpleThings/EntityAudit/AuditReader.php]: is this already used in this class?

Comment (TomasVotruba) [inline src/SimpleThings/EntityAudit/AuditReader.php]: @OskarStark Good point. #154

Comment (OskarStark) [inline src/SimpleThings/EntityAudit/AuditReader.php]: :smile:

**Pull request merged**